radixdlt/babylon-wallet-ios
# ABW-3715 Disable Accounts with Insufficient Funds

Jira ticket: [ABW-3715](https://radixdlt.atlassian.net/browse/ABW-3715)

## Description
When selecting fee payer accounts, we already filter out accounts with no XRD at all. We should also disable ones that don't have enough to pay the transfer and the fee. And show the "Insufficient balance" error.

### Notes
Additional notes.

## How to test
- Make sure there is an account with a very small but non-zero XRD balance, say 0.1
- Make a transfer in the app, from another account
- Try to change the fee payer
-> On the selection screen, the low balance account should be disabled for selection

- Also try transferring an amount of XRD from an account that is just above the balance of that account. Enough to cover the transfer (or the fee alone), but not the fee AND the transfer. 
-> Should lead to same result

## Screenshot

![image](https://github.com/user-attachments/assets/602c0379-98a0-4760-99da-93e5081cc300)

## PR submission checklist


[ABW-3715]: https://radixdlt.atlassian.net/browse/ABW-3715?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
